SPECS/vitess/vitess.spec

```diff
@@ -0,0 +1,153 @@
+# Generated by go2rpm
+%bcond_without check
+
+Name:           vitess
+Version:        8.0.0
+Release:        3%{?dist}
+Summary:        Database clustering system for horizontal scaling of MySQL
+# Upstream license specification: MIT and Apache-2.0
+License:        MIT and ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/vitessio/vitess
+#Source0:       https://github.com/vitessio/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/vitessio/vitess/archive/refs/tags/%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2021-04-26 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#
+Source1:        %{name}-%{version}-vendor.tar.gz 
+# To use with newer k8s
+Patch0:         0001-Add-context-to-k8s-calls.patch
+# To use with newer azure-storage-blob
+Patch1:         0001-Fix-for-newer-azure-storage-blob.patch
+# Fix unit test error
+Patch2:         0001-Fix-unit-test-error.patch
+Patch3:         update-go-module-version.patch
+BuildRequires: golang
+
+%description
+Vitess is a database clustering system for horizontal scaling of MySQL through
+generalized sharding.
+
+By encapsulating shard-routing logic, Vitess allows application code and
+database queries to remain agnostic to the distribution of data onto multiple
+shards. With Vitess, you can even split and merge shards as your needs grow,
+with an atomic cutover step that takes only a few seconds.
+
+
+%prep
+%setup -q
```

Comment:
Could you change this to `%autosetup -p1` instead and remove the `%patchX` macros?

Reply:
Fixed